base/applications/taskmgr/procpage.c

```diff
@@ -588,113 +636,89 @@ BOOL PerfDataGetText(ULONG Index, ULONG ColumnIndex, LPTSTR lpText, ULONG nMaxCo
     }
     if (ColumnDataHints[ColumnIndex] == COLUMN_MEMORYUSAGE)
```

Comment:
It would be better to use `switch`-`case`.

Reply:
It's not the purpose of this PR, but you're right. I'll change the design accordingly

Reply:
for later.

Reply:
Yes, implementation under test. Commit to follow.